hydra/core/utils.py

```diff
@@ -118,17 +118,23 @@ def run_job(
 
     old_cwd = os.getcwd()
     orig_hydra_cfg = HydraConfig.instance().cfg
-    HydraConfig.instance().set_config(config)
+
     working_dir = str(OmegaConf.select(config, job_dir_key))
     if job_subdir_key is not None:
         # evaluate job_subdir_key lazily.
         # this is running on the client side in sweep and contains things such as job:id which
         # are only available there.
         subdir = str(OmegaConf.select(config, job_subdir_key))
         working_dir = os.path.join(working_dir, subdir)
+
+    with read_write(config.hydra.runtime):
+        with open_dict(config.hydra.runtime):
+            config.hydra.runtime.output_dir = os.path.abspath(working_dir)
```

Comment:
I have a suggestion: How about renaming the `working_dir` variable to `output_dir` within the scope of this function body?
Here is my motivation for this suggestion: I think that the old `working_dir` name has become confusing now that `hydra.job.chdir` can be enabled/disabled; usually in compsci "working directory" refers to the current directory (i.e. pwd), but now this might not be the case. The if-block that appears on line 167 below looks like this currently:
```
        if _chdir:
            os.chdir(working_dir)
            ret.working_dir = working_dir
        else:
            ret.working_dir = "."
```
If renamed, it would look like this:
```
        if _chdir:
            os.chdir(output_dir)
            ret.working_dir = output_dir
        else:
            ret.working_dir = "."
```
And I feel that the latter is less confusing.

Reply:
fixed!